index.html

```diff
@@ -74,14 +74,21 @@
           <div class="row">
             <div class="col-12">
 
-              <!-- Nasa-HackUVic Logo -->
-              <figure class="img">
+              <!-- Nasa-HackUVic Logo
+                The order of the g path MUST be written as it is below. This is due to SVGs not being NICE and allowing
+                the use of the z-index property. SVGs calculate their z-index based on how the DOM renders the parts of
+                the SVGs. This is why the Swooshes and the HackUVic words and everything are split into multiple parts
+                and seem very disconnected when you're reading them straight on. I sold my soul for this knowledge.
```

Comment:
Thnank you for your soul contribution XD

Reply:
<3 <3 <3 <3 <3
It was worth it? Thank you for your appreciation. :'(